### ISSUE TITLE
ModuleNotFoundError: No module named 'torchcv'

### ISSUE THREAD

**Issue by hxy1051653358** [kuangliu/torchcv#49]
When I run python train.py
Traceback (most recent call last):
  File "train.py", line 19, in <module>
    from torchcv.models.fpnssd import FPNSSD512
ModuleNotFoundError: No module named 'torchcv'

**Comment by abhishekag03** [comment]
Yeah that will come because the directory containing torchcv is not a part of your system path. You can add it as follows:
import sys
sys.path.insert(0, "path_to_the_directory_containing_torchcv")
It will work now, you can check the directories added by "print(sys.path)"
You might need to add some more paths as you run the code.
